bin/dartdoc.dart

```diff
@@ -97,8 +97,15 @@ void _printHelpAndExit(ArgParser parser, {int exitCode: 0}) {
   _printUsageAndExit(parser, exitCode: exitCode);
 }
 
+/// Print usage information on invalid command lines.
 void _printUsageAndExit(ArgParser parser, {int exitCode: 0}) {
   print('Usage: dartdoc [OPTIONS]\n');
   print(parser.usage);
   exit(exitCode);
 }
+
+/// Print version information.
+void _printVersionAndExit(ArgParser parser, {int exitCode: 0}) {
```

Comment:
is `exitCode` ever passed in by clients?

Reply:
removed